app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -18,10 +18,13 @@
   </div>
 </div>
 
-<%= render SubnavComponent.new do |component| %>
-  <% @cohorts.each do |cohort| %>
-    <%= component.nav_item(path: lead_providers_your_schools_path(selected_cohort_id: cohort.id)) do %>
-      <%= cohort.start_year %> cohort
+<% if @cohorts.length > 1 %>
+  <%= render SubnavComponent.new do |component| %>
+    <% @cohorts.each do |cohort| %>
+      <%# The link to the current cohort doesn't have it in the URL %>
```

Comment:
Mhmmm, if we care about it enough to make a cryptic comment about it, should we add an automated test for it (cypress?)

Reply:
is the comment cryptic? 😅

Reply:
A little bit. Doesn't have it in the path param? Doesn't have it as a query param? Is that documenting a bug or intended outcome? 

Those are the first few questions I have when reading it :D

Reply:
it's a query param - i'll make the comment a bit clearer